src/pages/developers/tbtc-security-model.md

```diff
@@ -0,0 +1,126 @@
+2. A **signer group** forms and puts down **150% the value of 1 BTC in ETH** as collateral.
+3. The signer group generates a Bitcoin wallet using a [threshold ECDSA protocol](https://www.binance.vision/security/threshold-signatures-explained).
+4. The user sends 1 BTC to the Bitcoin wallet.
+5. The user generates an **SPV proof** of their deposit transaction, and sends it to the Ethereum chain.
+6. The user mints TBTC.
+
+Redeeming TBTC back into BTC is simply the same process in reverse:
+1. User burns TBTC and provides a receiving Bitcoin address to begin redemption.
+2. The **signer group** generates and publishes a signature for a Bitcoin transaction to send the Bitcoin to the user’s requested Bitcoin address.
+3. The user broadcasts this transaction and the generated signature to the Bitcoin network.
+TheI4.  signers generate an **SPV proof** of this transaction, publish it to the Ethereum chain, and receive their bonds back.
+
+There are three systems at play in this operation:
+
+1. The Keep Network, which creates signer federations per deposit to custody bitcoin.
+2. The Bitcoin price feed, which is used to determine the collateralisation of signers on the Ethereum host chain.
+3. The Bitcoin relay, which verifies BTC transactions to mint and redeem TBTC.
+
+## A decentralized network of signers
+
+tBTC uses the [Keep Network](https://keep.network/) to provide an always-online, decentralized network of signers. Each signer group that custodies a bitcoin deposit is made up of 3 nodes. These nodes are randomly selected from a large pool, using the [Keep random beacon](https://docs.keep.network/random-beacon/). Any opportunity to manipulate the signer set is mitigated by the KEEP token’s staking model, which provides Sybil resistance.
+
+To ensure users are compensated in the case of failure or fraud, signers are overcollateralized in ETH to a ratio of 150% the value of the bitcoin deposit. tBTC handles signer failure and signer fraud, punishing signers and fully reimbursing the user in TBTC, as well as making the signer ineligible for future work.
+
+### Signer failure
+
+Signers not responding to a redemption request in a timely manner constitutes a signer failure. Signer failures can include a failure to produce a signature for redemption, as well as failure to produce an SPV proof of inclusion in a Bitcoin block for a redemption.
+
+All failures are treated as protocol aborts, and initiate the [abort liquidation flow](https://docs.keep.network/tbtc/#redemption). Signer bonds are seized and auctioned off for TBTC so that the user can be reimbursed for their inaccessible funds, and half of the remainder of the signer bonds (a maximum of 1/6th of the original bond) are returned to the signers.
+
+In order to recoup their losses from liquidation, signing groups may choose to recover BTC after an abort has commenced. The reference client leaves offline coordination to each signer.
```

Comment:
how do the signers coordinate to do the recovery if the identities are hidden to prevent collusion?

Reply:
> the identities are hidden

Could you point to where you've got this from? Identities of signers (Ethereum addresses), are public.